usagov/usagov-2021
# usagov-1355 remove searchbar clear button

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-

## Description

Remove the clear icon (the x that appears when you start typing in a search field) from the searchbar components.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- There are 4 search bars on the site you can test. 1 & 2) The two headers (desktop and mobile) 3& 4) Federal agencies index (English and Spanish) 
- When you start typing you shouldn't see the x icon anymore 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
